packages/pn-commons/src/components/SideMenu/SideMenu.tsx

```diff
@@ -83,10 +83,12 @@ const SideMenu: FC<Props> = ({ menuItems, selfCareItems }) => {
     setState(!state);
   };
 
-  const handleNavigation = (item: SideMenuItem, menuFlag?: boolean) => {
-    if (isMobile && !menuFlag) {
```

Comment:
First to do these changes, understand what they do.
Understand why we have isMobile and menuFlag checks